conf/inria.yaml

```diff
@@ -21,7 +21,7 @@ experiment:
     learning_rate_schedule_patience: 6
     in_channels: 3
     num_classes: 2
-    ignore_index: 0  # class 0 not used for scoring
+    ignore_index: null
```

Comment:
@calebrob6 can you double check this? It relates to #644/#444.

Reply:
looks good to me